UCI-HyperXite/hx9-control-system
# Add `SignalLight` component and demo with RPPAL

## Changes
- Install RPPAL for using Rasperry Pi peripherals
- Add `SignalLight` component with blinking demo

## Testing
1. Compile Rust project for the Raspberry Pi
2. Transfer the `pod-operation` executable to the Raspberry Pi on the pod
3. Confirm GPIO pin 21 is connected to the relay hat used to control the signal light
4. Run the pod operation program
5. Observe the signal light switches between on and off
